src/main/java/frc/robot/subsystems/swerve/SwerveModuleIoTalonFx.java

```diff
@@ -87,14 +87,17 @@ public SwerveModuleIoTalonFx(WheelModuleIndex index) {
 
     // Set Drive TalonFXConfiguration.
     var driveTalonFXConfiguration = new TalonFXConfiguration();
-    driveTalonFXConfiguration.CurrentLimits.StatorCurrentLimit = 40.0;
-    driveTalonFXConfiguration.CurrentLimits.StatorCurrentLimitEnable = true;
+    var currentLimits = driveTalonFXConfiguration.CurrentLimits;
+    currentLimits.StatorCurrentLimit = 40.0;
+    currentLimits.StatorCurrentLimitEnable = true;
+    currentLimits.SupplyCurrentThreshold = 80.0;
+    currentLimits.SupplyTimeThreshold = 0.5;
     driveMotor.getConfigurator().apply(driveTalonFXConfiguration);
```

Comment:
I think we also need to set `SupplyCurrentLimitEnable` to `true`.

Reply:
Good catch, it seems like we will need to just use supply current if we want both continuous and momentary limits.